vassal-app/src/main/java/VASSAL/build/module/Map.java

```diff
@@ -2154,9 +2205,9 @@ else if (py < vrect.height && py >= vrect.height - dist) {
        */
       @Override
       public void timingEvent(float fraction) {
-        // Constant velocity along each axis, 0.5px/ms
+        // Constant velocity along each axis, 0.5px/ms default
         final long t1 = System.currentTimeMillis();
-        final int dt = (int)((t1 - t0) / 2);
+        final int dt = (int)((t1 - t0) * NumberUtils.max(EDGE_SCROLL_RATE, 0.01));
```

Comment:
Is there a reason why `Math.max()` doesn't work here?

Reply:
It will probably work fine, I guess I just got on a tear of using NumberUtils

Reply:
(I did change it to Math, in case that wasn't clear)